backend/templates/emails/ticket_transfer.html

```diff
@@ -0,0 +1,27 @@
+<!-- TYPES:
+sender_first_name:
+    type: string
+receiver_first_name:
+    type: string
+receiver_username:
+    type: string
+event_name:
+    type: string
+type:
+    type: string
+-->
+
+{% extends 'emails/base.html' %}
+
+{% block content %}
+<h2>Thanks for using Penn Clubs!</h2>
+
+<p style="font-size: 1.2em">
+    {{ sender_first_name }}, this is confirmation that you have transferred a ticket to {{ receiver_first_name }} ({{ receiver_username }}) for {{ event_name }} with ticket type {{ type }}.
+</p>
```

Comment:
nit: would be nice if we could throw in a link to the ticket as an href.

Reply:
We also send the new owner of the ticket a confirmation email, which should contain a QR code that has a link to the ticket

Reply:
Checks out, I'll leave this up to you.